src/transformers/models/gpt_neo/modeling_gpt_neo.py

```diff
@@ -192,6 +192,57 @@ def _look_back(tensor, block_length, window_size, pad_value=0, is_key_value=True
             padded_tensor = padded_tensor.transpose(-2, -1)
         return padded_tensor
 
+    @staticmethod
+    def _split_seq_length_dim_to(tensors, dim_factor_1, dim_factor_2, hidden_size):
```

Comment:
(nit) split to me always sounds like a tensor is divided into two (substraction) whereas here it is reshaped to have one more dimension (division). I would probably rather call it `_factor_seq_length_dim_to(...)` or `_divide_seq_...`

Reply:
2nd nit, think it's a bit cleaner to not pass `hidden_size` here

Reply:
Here seq length dimension is actually split into two, precisely into `(num_blocks, block_length)`